atom/browser/native_window_views_win.cc

```diff
@@ -207,9 +207,14 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
       return false;
     }
     case WM_MOVING: {
-      if (!movable_)
+      bool prevent_default = false;
+      NotifyWindowWillMove(gfx::Rect(*reinterpret_cast<RECT*>(l_param)),
+                           &prevent_default);
+      if (!movable_ || prevent_default) {
         ::GetWindowRect(GetAcceleratedWidget(),
                         reinterpret_cast<RECT*>(l_param));
+        return true;
```

Comment:
What does the changed return value do here?

For the most part this patch seems symmetrical with the `WM_SIZING` handler except that it always returns false.

Reply:
Without the change in value, frameless windows can still be moved with -webkit-app-region drag

Reply:
Makes sense. Could you add a short comment in the code to this effect?

Reply:
@ckerr  done